src/main/java/com/vectorcast/plugins/vectorcastexecution/VectorCASTSetup.java

```diff
@@ -429,6 +446,33 @@ private void processDir(File dir, String base, FilePath destDir, Boolean directD
             }
         }
     }
+    
+    public void printVersion(TaskListener listener, JarFile jFile) {

```

Comment:
Looks like this contains duplicate code as `NewPipelineJob::getVersion`.
Can we move `NewPipelineJob::getVersion` outside of `NewPipelineJob` and have `NewPipelineJob` and this function call it?

Reply:
If you can find a better/safer way to implement, that's awesome. Please note that there is a similar block inside NewPipelineJob.java.